css/css-values/getComputedStyle-border-radius-001.html

```diff
@@ -46,28 +46,28 @@
 
   var targetElement = document.getElementById("target");
 
-    function verifyComputedStyle(property_name, expected_value, description)
+    function verifyComputedStyle(property_name, expected_value)
     {
 
     test(function()
       {
 
       assert_equals(getComputedStyle(targetElement)[property_name], expected_value);
 
-      }, description);
+      }, `testing ${property_name}: ${specified_value}`);
```

Comment:
From https://wpt.fyi/results/css/css-values/getComputedStyle-border-radius-001.html?diff&filter=ADC&run_id=6195533447692288&run_id=5201256571731968 (click "Show Details"): `Uncaught ReferenceError: specified_value is not defined`
```suggestion
      }, `testing ${property_name}: ${expected_value}`);
```

Reply:
Resolved in [ba1dbe2](https://github.com/web-platform-tests/wpt/pull/42517/commits/ba1dbe2ef52b84e47bcf6ecf34f6059769bf44a8)